cluster-autoscaler/simulator/cluster.go

```diff
@@ -294,6 +294,10 @@ func findPlaceFor(removedNode string, pods []*apiv1.Pod, nodes map[string]bool,
 		return fmt.Sprintf("%s/%s", pod.Namespace, pod.Name)
 	}
 
+	nodeExists := func(nodeName string) bool {
```

Comment:
It's not really what the function is checking - the nodes you're filtering out do exist. Maybe rename it to isCandidateNode or something along those lines?

Reply:
Done